code/modules/mob/living/simple_animal/hostile/megafauna/drake.dm

```diff
@@ -4,6 +4,9 @@
 #define SWOOP_DAMAGEABLE 1
 #define SWOOP_INVULNERABLE 2
 
+#define DRAKE_FIRE_TEMP 500
+#define DRAKE_FIRE_EXPOSURE 50
```

Comment:
please document your code according to our [contribution standards](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md#documenting-a-variabledefine)

Reply:
document my nuts

Reply:
no

Reply:
![image](https://user-images.githubusercontent.com/10467687/85349207-43190100-b4d4-11ea-8f9a-cf60631a3d6d.png)

Reply:
```suggestion
/// Used whenever the drake generates a hotspot
#define DRAKE_FIRE_TEMP 500
/// Used whenever the drake generates a hotspot
#define DRAKE_FIRE_EXPOSURE 50
```

Reply:
only because bobbahbrown said I didn't know how to add comments